vault_install.sh

```diff
@@ -1,7 +1,15 @@
 #!/bin/bash
 
 # This script updates the package list, installs Gpg, downloads the Hashicorp signing key, adds the Hashicorp repository and installs Vault.
-sudo apt update
+
+# Update Apt
+if (( $(date +%s) - $(stat -c %Y /var/lib/apt/periodic/update-success-stamp) > 24*60*60 ))
```

Comment:
What are you doing here?

Reply:
I looked at several ways to check if apt had been updated. This version seemed to be popular on stack overflow. Folks were commenting that the cache and apt/lists weren’t reliable methods to determine if it had been updated but idk.